src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -22,6 +22,8 @@
                     "value": "Multiple Choice: {{ mcCount }}, Drag-and-Drop: {{ dndCount }}, Kurzantwort: {{ shortCount }}"
                 }
             },
+            "create": "Quiz erstellen",
+            "import": "Quiz importieren",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys